makefile

```diff
@@ -76,6 +76,20 @@ test-frontend-e2e-dev:
 test-frontend-e2e-prod:
 	(cd frontend; \
 	ENVIRONMENTS=prod ${TEST_FRONTEND_CMD})
+## - codegen: "codeless" Playwright generation of UI tests by recording browser interaction
+##   You don't need to use 'local' necessarily if you want to write the tests for local. Any of these commands can work
+##   to write tests that are theoretically compatible in any environment (as long as the UI is the same). The only
+##   difference is that when the test code is written, the first line will hard-code the URL to that environment. So
+##   after recording, the code should be repuprosed in the style of frontend/tests/.
+codegen-local:
```

Comment:
## New "codeless test generation" commands
I added some documentation on how to use them and what they are in the `makefile`.